### DISCUSSION TITLE
revert to last position on reopening

### DISCUSSION THREAD

**Discussion by neripunk** [vedang/pdf-tools#217] [Ideas]
Hi, I've started playing around with pdf-tools and I think it's amazing, thanks a lot for your work.

I did have one feature suggestion; I have save-place-mode enabled in my emacs so files reopen in the same position after being closed. However, this does not seem to work for pdfs. It would be excellent if this could be implemented, especially for reading through longer documents.

Thanks!

**Comment by port19x** [comment]
Bookmarks work for saving the position, that's how I get around that limitation
